### ISSUE TITLE
A11y_AzureDataStudioAAE_Sql Instance 1_Compute + Storage_ColorContrast: Color contrast ratio of Keyboard focus outline on "scaling computes vCores and memory" link is less than 3:1

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15741]
![15741_A11y_AzureDataStudioAAE_Sql Instance 1_Compute + Storage_ColorContrast](https://user-images.githubusercontent.com/85863543/127596597-75213f1d-ea3e-4126-9abc-fc6cdd28f703.png)
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2; #A11yWCAG2.1;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.11;#AINotInScope;#ColorContrast;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sql Instance 1" and open manage and tab to "Compute + Storage" page and press enter
5. Tab to "scaling computes vCores and memory" link
6. Start Color Contrast analyser(CCA) tool and analyse color contrast ratio of keyboard focus outline on "scaling computes vCores and memory" link

### Actual:
Color contrast ratio of Keyboard focus outline on "scaling computes vCores and memory" link is less than 3:1

Same issue is also observed for the scenario given below:

1. PG1_Overview(Dashboard)
2. PG1_Properties
3. PG1_Connection Strings
4. PG1_Compute + Storage
5. PG1_Coordinator Node Parameter
6. PG1_Worker Node Parameter

### Expected:
Color contrast ratio of Keyboard focus outline on "scaling computes vCores and memory" link should be more than or equal to 3:1

### User Impact:
Low vision users will not be able to see keyboard focus properly if color contrast if low.

**Comment by krrish2021** [comment]
@candiceye the issue still repro in in latest azure data studio @Charles-Gagnon Please reopen the bug
![Screenshot (177)](https://user-images.githubusercontent.com/85863543/150953845-0621a031-58b4-4050-82aa-cf7da4d467f3.png)

**Comment by krrish2021** [comment]
Issue is fixed and it does not repo.
![Screenshot (214)](https://user-images.githubusercontent.com/85863543/151787831-503590be-c5c9-4851-8207-b27e42121904.png)

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
